options_native.go

```diff
@@ -0,0 +1,52 @@
+// +build !js
+
+package websocket
+
+import (
+	"crypto/tls"
+	"net/http"
+
+	"github.com/gorilla/websocket"
+)
+
+// WebsocketConfig is the configuration type for the Websocket transport.
+type WebsocketConfig struct {
```

Comment:
If this is an exported struct, we don't need the Option constructor (and vice versa).